src/NUnitConsole/nunit3-console/Program.cs

```diff
@@ -64,6 +65,20 @@ public static int Main(string[] args)
                 return ConsoleRunner.INVALID_ARG;
             }
 
+            if (!string.IsNullOrEmpty(Options.ConsoleEncoding))
+            {
+                try
+                {
+                    Console.OutputEncoding = Encoding.GetEncoding(Options.ConsoleEncoding);
```

Comment:
The default encoding for Windows CMD is [CP437](https://en.wikipedia.org/wiki/Code_page_437), which is very limited. Changing this to an alternate encoding (most often UTF-8) will allow a much broader range of characters that can be printed.

For my particular use case, I am printing a large amount of output from each test to our CI server, which catalogs all test runs & output (automated Selenium testing). The special characters are used to improve the readability of this output for review by non-developers.